src/components/DepositWidget/Table.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Row, RowProps } from './Row'
+
+const TableWrapper = styled.table`
+  display: table;
+  flex-direction: colum;
```

Comment:
It's already a table, why display is table? 
Also, flex-direction is not for flex displays?

Reply:
CSS copy/paste from other style. Will remove.